src/aiortc/codecs/g711.py

```diff
@@ -69,6 +71,10 @@ def encode(
         data = self._convert(data, SAMPLE_WIDTH)
         return [data], timestamp
 
+    def pack(self, packet: Packet) -> Tuple[List[bytes], int]:
```

Comment:
It works? How can test g711 codec?

Reply:
I haven't yet managed to get it to work, because the payload I get are too large to fit in an RTP packet. That's why the title of this ticket is about Opus